application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -3328,6 +3345,7 @@ enum RelativeDirection {
   CONTINUE
```

Comment:
Add description for this enum value. Maybe we could add descriptions for some other values in this enum as well as not all of them are clear.